File: Dominio/ObjetosDeValor/Embarcador/Enumeradores/EnumVersaoLayoutATM.cs
namespace Dominio.ObjetosDeValor.Embarcador.Enumeradores
{
    public enum EnumVersaoLayoutATM
    {
        Versao200 = 1,
        Versao400 = 2
    }

    public static class EnumVersaoLayoutATMHelper
    {
        public static string Descricao(this EnumVersaoLayoutATM versaoLayoutATM)
        {
            switch (versaoLayoutATM)
            {
                case EnumVersaoLayoutATM.Versao200:
                    return "Versão 2.00";
                case EnumVersaoLayoutATM.Versao400:
                    return "Versão 4.00";
                default:
                    return string.Empty;
            }
        }
    }
}

File: SGT.WebAdmin/ViewsScripts/Enumeradores/EnumVersaoLayoutATM.js
let EnumVersaoLayoutATMHelper = function () {
    this.Versao200 = 1;
    this.Versao400 = 2;
};

EnumVersaoLayoutATMHelper.prototype = {
    obterOpcoes: function () {
        return [
            {
                value: this.Versao200,
                text: "Versão 2.00"
            }, {
                value: this.Versao400,
                text: "Versão 4.00"
            }
        ];
    }
}

let EnumVersaoLayoutATM = Object.freeze(new EnumVersaoLayoutATMHelper());